tests/cypress/support/helpers.js

```diff
@@ -81,231 +81,103 @@ const convertDateToDisplayDate = (date) => {
   return `${dateString} between ${timeString}`
 }
 
-const navigateToCommunalPage = () => {
-  cy.visit('http://localhost:3000/report-repair/');
-  makeSelectionAndClickButton('Something else');
-}
-
-const navigateToPostcodePage = () => {
+function navigateToReportRepair() {
   cy.visit('http://localhost:3000/report-repair/');
-  makeSelectionAndClickButton('Something else');
-  cy.get('[data-cy=communal]', {timeout: 10000}).then(($loadedSection) => {
-    makeSelectionAndClickButton('No');
-  });
-  cy.get('[data-cy=postcode]', {timeout: 10000})
-}
-
-const navigateToAddressPage = () => {
-  cy.visit('http://localhost:3000/');
-  cy.contains('Start now').click();
-  cy.contains('Something else').click();
-  cy.get('button').click();
-  cy.get('[data-cy=communal]', {timeout: 10000}).then(($loadedSection) => {
-    cy.contains('No').click();
-  });
+  cy.get('[data-cy=priority-list]', {timeout: 10000})
 }
 
-const navigateToBestDescriptionPage = () => {
-  cy.visit('http://localhost:3000/report-repair/');
```

Comment:
I've noticed that `localhost:3000/report-repair` is used 29 times across the cypress tests. Maybe it would be worth putting it in a variable? `baseUrl` or something like that?

Reply:
This could be expanded to be a helper method that would also validate the page has loaded (with a `cy.wait`) and should be addressed separately.

Reply:
This has been addressed in PR #65.